### PR TITLE
Unified sample for audio and video calling

### PR DESCRIPTION
**Pull request by jowang-msft**
## Purpose

* Consolidate the audio and video call into one example (UWP)

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[x] Feature
[x] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone Azure-Samples/communication-services-dotnet-quickstarts
cd communication-services-dotnet-quickstarts
git checkout main
npm install
```

* Test the code
```
  * Load CallingQuickstart.csproj in Visual Studio 2019
  * Acquire a validation ACS calling token and replace <AUTHENTICATION_TOKEN> in InitCallAgentAndDeviceManagerAsync
  * Build and run the project to initiate a voice/video call to another ACS party provisioned in the same ACS subscription
```

## What to Check
Verify that the following are valid
* Make sure webcam and microphone permission is granted first time running the sample

### CODE REVIEW

**Review by lucianopa-msft — APPROVED** [review]
LGTM!

File: VoiceCalling/Readme.md [hunk 1]
```diff
@@ -1,13 +0,0 @@
-## Prerequisites
```
Comment (viniciusl-msft) [inline VoiceCalling/Readme.md:1]: Removing this was intended? Did something change on the instructions?

File: Calling/Package.appxmanifest [hunk 2]
```diff
@@ -1,50 +1,60 @@
...
+  <Capabilities>
+    <Capability Name="internetClient" />
+    <DeviceCapability Name="webcam"/>
+    <DeviceCapability Name="microphone"/>
+  </Capabilities>
+
+  <Extensions>
+    <Extension Category="windows.activatableClass.inProcessServer">
+      <InProcessServer>
+	    <Path>RtmMvrUap.dll</Path>
```
Comment (viniciusl-msft) [inline Calling/Package.appxmanifest:55]: Would customers be able to refer this dll directly? Should they?

File: Calling/Package.appxmanifest [hunk 3]
```diff
@@ -1,50 +1,60 @@
...
+  <Identity
+    Name="646c0233-13f7-4ea3-b02e-703b86c51a27"
+    Publisher="CN=jowang"
+    Version="1.0.0.0" />
+
+  <mp:PhoneIdentity PhoneProductId="646c0233-13f7-4ea3-b02e-703b86c51a27" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
+
+  <Properties>
+    <DisplayName>Calling</DisplayName>
+    <PublisherDisplayName>jowang</PublisherDisplayName>
```
Comment (viniciusl-msft) [inline Calling/Package.appxmanifest:18]:
```suggestion
    <DisplayName>Azure Communication Calling</DisplayName>
    <PublisherDisplayName>Microsoft</PublisherDisplayName>
```

File: Calling/Package.appxmanifest [hunk 4]
```diff
@@ -1,50 +1,60 @@
...
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="646c0233-13f7-4ea3-b02e-703b86c51a27"
+    Publisher="CN=jowang"
```
Comment (viniciusl-msft) [inline Calling/Package.appxmanifest:11]: I don't know what's the common suggestion here, but maybe we can consider:
```suggestion
    Name="646c0233-13f7-4ea3-b02e-703b86c51a27"
    Publisher="CN=Microsoft"
```

**Comment by DaybreakQuip** [comment]
When I call someone, then hang up, then call again, my local video becomes blank. Is that a problem within our SDK or does the test app need to have a handle for that?

**Review by viniciusl-msft — CHANGES_REQUESTED** [review]

File: Calling/Package.appxmanifest (hunk 3 above)
Comment (jowang-msft) [reply Calling/Package.appxmanifest:18]: Oops, missed this :)

File: Calling/Package.appxmanifest (hunk 4 above)
Comment (jowang-msft) [reply Calling/Package.appxmanifest:11]: Will update, thanks.

**Comment by jowang-msft** [comment]
> When I call someone, then hang up, then call again, my local video becomes blank. Is that a problem within our SDK or does the test app need to have a handle for that? Or for the quickstart, would this not matter, because the first call, everything works fine.

Yeah, this is likely a XAML/MediaElement issue. ME disappeared on it's 2nd assignment to its Source property. I kinda workaround it without reassigning, it should work better now. Although there still seems to be issues when repeating call/hangup many times, I think it's less of an issue for the purpose of this sample code.

**Review by DaybreakQuip — APPROVED** [review]

**Review by viniciusl-msft — APPROVED** [review]

**Comment by jowang-msft** [comment]
#sign-off

**Review by gaobob-msft — APPROVED** [review]

**Pull request merged**